backend/api/v1/sql_service.go

```diff
@@ -1289,8 +1289,8 @@ func convertAdviceList(list []*storepb.Advice) []*v1pb.Advice {
 			Code:    int32(advice.Code),
 			Title:   advice.Title,
 			Content: advice.Content,
-			Line:    int32(advice.GetStartPosition().Line),
```

Comment:
Is it possible to make `Line` and `Column` private to prevent similar bugs?

Reply:
They are generated by Protobuf, because of the need for marshaling(using reflect), export(uppercase) is required. It might be better to use linter, but I didn't find an out-of-the-box configuration.